src/service/performance-impl.js

```diff
@@ -670,23 +649,17 @@ export class Performance {
 
   /**
    * Tick the largest contentful paint metrics.
```

Comment:
Nit: would be nice to move the earlier comment about why we use `startTime` to here

Reply:
Done.